src/modules/user/user.usecase.ts

```diff
@@ -698,6 +700,72 @@ export class UserUseCases {
     });
   }
 
+  async sendAccountUnblockEmail(email: User['email']): Promise<void> {
+    const secret = this.configService.get('secrets.jwt');
+    const user = await this.userRepository.findByEmail(email);
+
+    if (!user) {
+      throw new NotFoundException();
+    }
+
+    const defaultIat = getTokenDefaultIat();
+
+    await this.userRepository.updateByUuid(user.uuid, {
+      lastPasswordChangedAt: new Date(defaultIat * 1000),
+    });
+
```

Comment:
I used defaultIat as reference because it is possible to lose some miliseconds when we round a number for the Iat. Thus, it is better to use it as reference to create the date.